src/pages/Content/common/DashboardService.tsx

```diff
@@ -36,7 +40,7 @@ export const renderDashboard = (componentsConf: ComponentConfig[]) => {
 
     const ele = insertElement();
     if (!elementExists(ele)) {
-      console.log('Error! Element not exist!')
+      console.log('Info: Element not exist!')
```

Comment:
Do we need a framework logger rather than `console.log`?

Reply:
Thanks @frank-zsy . I think `console.log` is enough for Chrome Extention. Maybe we can also make a simple wrapper for `console.log`, `console.info` and `console.error`.